### DISCUSSION TITLE
How to convert or can we convert detectron2 to ONNX with python inferencing ? Is there an update regarding this ?

### DISCUSSION THREAD

**Discussion by monisha98reddy** [facebookresearch/detectron2#4163] [Q&A]
**How to convert or can we convert detectron2 to ONNX with python inferencing ?**

**Comment by thiagocrepaldi** [comment]
I am working on adding support to ONNX export (with and without caffe2) https://github.com/facebookresearch/detectron2/pull/4120 and https://github.com/facebookresearch/detectron2/pull/4153/

Hopefully it gets accepted soon
